modules/reindex/src/test/java/org/elasticsearch/index/reindex/RoundTripTests.java

```diff
@@ -56,11 +56,27 @@ public void testReindexRequest() throws IOException {
         randomRequest(reindex);
         reindex.getDestination().version(randomFrom(Versions.MATCH_ANY, Versions.MATCH_DELETED, 12L, 1L, 123124L, 12L));
         reindex.getDestination().index("test");
+        if (true) {
```

Comment:
Sorry! That is a leftover from a serialization bug I was hunting down. I'll push a fix.